go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	github.com/spf13/cast v1.5.1
 	github.com/zclconf/go-cty v1.14.1
 	go.mongodb.org/atlas v0.35.0
-	go.mongodb.org/atlas-sdk/v20231001001 v20231001001.1.0
+	go.mongodb.org/atlas-sdk/v20231001002 v20231001002.0.0
```

Comment:
It does not seems to use SDK preview. 🤔 what am I missing?

Reply:
correct, this is the latest published version

Reply:
I don't understand why this PR is linked to the PR that talks about SDK preview

Reply:
sorry, the other PR was mainly to fix the SDK Update, And additionally I added the possibility to use the preview sdk locally, never merging it

Reply:
the preview can be used locally without the need to open a PR in terraform